apps/meteor/app/livechat/server/lib/RoutingManager.ts

```diff
@@ -65,47 +65,27 @@ export const RoutingManager: Routing = {
 	methodName: null,
 	methods: {},
 
-	startQueue() {
-		// todo: move to eventemitter or middleware
-		// queue shouldn't start on CE
-	},
-
-	isMethodSet() {
-		return !!this.methodName;
-	},
-
-	async setMethodNameAndStartQueue(name) {
-		logger.info(`Changing default routing method from ${this.methodName} to ${name}`);
-		if (!this.methods[name]) {
-			logger.warn(`Cannot change routing method to ${name}. Selected Routing method does not exists. Defaulting to Manual_Selection`);
-			this.methodName = 'Manual_Selection';
-		} else {
-			this.methodName = name;
-		}
-
+	async startQueue() {
 		const shouldPreventQueueStart = await License.shouldPreventAction('monthlyActiveContacts');
 
 		if (shouldPreventQueueStart) {
 			logger.error('Monthly Active Contacts limit reached. Queue will not start');
 			return;
 		}
-
 		void (await Omnichannel.getQueueWorker()).shouldStart();
 	},
 
+	isMethodSet() {
+		return settings.get<string>('Livechat_Routing_Method') !== '';
```

Comment:
By changing this we are not using `methodName` anymore. Is this the expected behavior? If so, let's remove that variable. I think here we need to revisit it too: https://github.com/RocketChat/Rocket.Chat/blob/1b50c634c9b035efe899cfedd27c2589bb3bc2f9/apps/meteor/app/livechat/server/lib/RoutingManager.ts#L96